src/App.js

```diff
@@ -1,23 +1,84 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { UserDetails, Birthday, Name, Home } from "./pages";
+
+import "./App.css";
+import { validateBirthday } from "./api";
 
 function App() {
+  const [name, setName] = useState({ first: "", last: "" });
```

Comment:
I feel like this this business logic lacks structure. Could we try to implement using a scalable pattern like [Flux](https://www.freecodecamp.org/news/an-introduction-to-the-flux-architectural-pattern-674ea74775c9/)?